support-frontend/assets/components/priceLabel/priceLabel.jsx

```diff
@@ -13,7 +13,7 @@ export type PropTypes = {
   billingPeriod: BillingPeriod,
 }
 const displayPriceForPeriod = (productPrice: Price, billingPeriod: BillingPeriod) =>
-  `${showPrice(productPrice)} Every ${billingPeriodNoun(billingPeriod)}`;
```

Comment:
this will change the casing in the dp checkout too, think it should be like this?